### PR TITLE
Run tests for SQLite on .NET Core

### PR DESCRIPTION
**Pull request by hazzik**

### CODE REVIEW

File: src/NHibernate.Test/NHibernate.Test.csproj [hunk 1]
```diff
@@ -64,14 +65,13 @@
...
     <PackageReference Include="Oracle.ManagedDataAccess" Version="12.1.2400" />
-    <PackageReference Include="System.Data.SQLite.Core" Version="1.0.106" />
     <PackageReference Include="MySql.Data" Version="6.9.11" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.0'">
     <PackageReference Include="System.CodeDom" Version="4.4.0" />
     <PackageReference Include="System.Data.SqlClient" Version="4.4.2" />
     <PackageReference Include="System.Data.OracleClient" Version="1.0.8" />
-    <PackageReference Include="System.Data.Odbc" Version="4.5.0-preview1-25914-04" />
+    <PackageReference Include="System.Data.Odbc" Version="4.5.0" />
```
Comment (fredericDelaporte) [inline src/NHibernate.Test/NHibernate.Test.csproj]: A bit unrelated but why not, a dedicated PR would be overkill.

**Comment by hazzik** [comment]
There is also a bug on Linux/MacOS X I’ve logged: https://system.data.sqlite.org/index.html/tktview?name=d292f2e23d

File: src/NHibernate/Dialect/SQLiteDialect.cs [hunk 2]
```diff
@@ -141,6 +141,7 @@ protected virtual void RegisterFunctions()
 			"longtext",
 			"memo",
 			"money",
+			"native",
```
Comment (fredericDelaporte) [inline src/NHibernate/Dialect/SQLiteDialect.cs]: According to the test failure on AppVeyor, the missing word was `nothing`, not `native`.

Comment (hazzik) [reply src/NHibernate/Dialect/SQLiteDialect.cs]: True. Does not work anyway.

**Comment by hazzik** [comment]
It's so sad being earliest adopters :(

**Comment by fredericDelaporte** [comment]
Visual Basic tests fail for SQLite. Maybe the project needs a direct reference.

**Comment by hazzik** [comment]
Yes, I saw this.

**Pull request merged**